src/components/TodoListItem.tsx

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable react/prop-types */
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import style from "./TodoListItem.module.css";
-const TodoListItem = (props) => {
+
+const TodoListItem = (props: { todo: Todo; onRemoveTodo: any }) => {
```

Comment:
in your types.ts file 

```
export interface TodoListItemProps {
    todo: tTtodolistItem
    onRemoveTodo : (id: string) => void
}
```

then in this file


```suggestion
const TodoListItem = (props: TodoListItemProps) => {
```